msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md

```diff
@@ -94,11 +156,11 @@ The following image displays the tab content with **Red** icon selected:
 
 Choosing the appropriate button triggers either `saveGray()` or `saveRed()`, and invokes the following:
 
-* Set `settings.setValidityState(true)` to true.
-* The `microsoftTeams.settings.registerOnSaveHandler()` event handler is triggered.
+* Set `pages.config.setValidityState(true)` to true.
+* The `app.pages.config.registerOnSaveHandler()` event handler is triggered.
```

Comment:
This should be `pages.config.registerOnSaveHandler` (remove 'app')

Reply:
Addressed in Michael's last commit